SYCL/DeprecatedFeatures/SpecConsts1.2.1/composite-type.cpp

```diff
@@ -1,7 +1,6 @@
 // UNSUPPORTED: cuda || hip
 //
 // RUN: %clangxx -fsycl -D__SYCL_INTERNAL_API %s -o %t.out
```

Comment:
Spec constant tests were removed in #1187

Reply:
Removed after merge commit.